internal/plugin/grpc_client_plugin.go

```diff
@@ -120,7 +130,8 @@ func (gc *GrpcClient) createConnection() error {
 		},
 	}
 
-	reqCtx, reqCancel := context.WithTimeout(context.Background(), gc.config.Common.MaxElapsedTime)
+	ctx := context.Background()
```

Comment:
not sure this change is necessary?

Reply:
its needed because I cant pass `reqCtx` to the Process function below. The `reqCtx` times out when I am sending the data plane status update